scope-hoisting-closure/excersice-2.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * 2.	Write	a	sum	function	which	will	 work	properly	when	invoked	using	syntax	below.
+ */
+
+function add(a, b) {
+  return a + b;
+}
```

Comment:
I would move this function inside function sum to keep global object clean. 
I believe this is best practice.

Reply:
Overall, great solution!